assets/js/src/core/components/versions-fields-list/components/data-component/data-component.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import React from 'react'
+import { useInjection } from '@Pimcore/app/depency-injection'
+import { type DynamicTypeObjectDataRegistry } from '@Pimcore/modules/element/dynamic-types/defintinitions/objects/data-related/dynamic-type-object-data-registry'
+import { serviceIds } from '@Pimcore/app/config/services/service-ids'
+import { Alert } from 'antd'
+import ErrorBoundary from '@Pimcore/modules/app/error-boundary/error-boundary'
+import { type ObjectComponentProps } from '@Pimcore/modules/data-object/editor/types/object/tab-manager/tabs/edit/components/object-component'
+
+export interface DataComponentProps extends ObjectComponentProps {
+  datatype: 'data'
+  fieldType?: string
+  [p: string]: any
+}
+
+export const DataComponent = (props: DataComponentProps): React.JSX.Element => {
+  const { fieldType } = props
+
+  const objectDataRegistry = useInjection<DynamicTypeObjectDataRegistry>(serviceIds['DynamicTypes/ObjectDataRegistry'])
+
+  const currentFieldType = fieldType ?? 'unknown'
+
+  if (!objectDataRegistry.hasDynamicType(currentFieldType)) {
+    return (
+      <Alert
+        message={ `Unknown data type: ${currentFieldType}` }
+        type="warning"
+      />
+    )
+  }
+
+  const objectDataType = objectDataRegistry.getDynamicType(currentFieldType)
+
+  const _props = {
+    ...props,
+    noteditable: true
```

Comment:
Wouldn't it be a better idea to set this noteditable: true directly in the default getVersionObjectDataComponent() of the abstract class instead?